### PR TITLE
Export functionality

### PR DESCRIPTION
**Pull request by Korab-Bejta**
We created a service that injects itself into fairchain.component and provides a download function which creates a txt file.

### CODE REVIEW

File: src/app/importExport.service.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+  })
```
Comment (mdeliso97) [inline src/app/importExport.service.ts:5]: Comment missing, what is this method responsible for? Also missing a general explanation of the service itself

**Review by mdeliso97 — CHANGES_REQUESTED** [review]
The overall implementation looks pritty interesting, it is a good start on the actual functionality we are looking for, well done with the pop up for download file. I wouldn't merge this on dev, as it is a too erly version, which doesn't work properly and needs more enhancements.
Look also the comment of a correction I posted below

File: src/app/importExport.service.ts (hunk 1 above)
Comment (LYCChiang) [reply src/app/importExport.service.ts:5]: This is an essential part of a service. In components, you have to import "component" from angular/core into the typescript file and write @component ({ selector: ...}) to make it an component. Likewise, in services you have to import "injectable" from angular/core and write the code above to make it a service. This enables the class to be injected into components.

Comment (mdeliso97) [reply src/app/importExport.service.ts:5]: Thanks for the explanation, can you make a comment in the service with a short explanation?

**Pull request closed without merging**